lib/jobs.rb

```diff
@@ -1,18 +1,28 @@
 module Jobs
-  SIMPLE_SEARCHES = '(job|employment|internship)s?'
-  JOB_RELATED_KEYWORDS = '((position|opening|posting|job|employment|intern(ship)?|seasonal|trabajo|puesto|empleo|vacante)s?|(opportunit|vacanc)(y|ies))|(posicion|ocupacion|oportunidad|federal)(es)?|gobierno'
-  SCRUB_KEYWORDS = JOB_RELATED_KEYWORDS.remove(/\|intern\(ship\)|\|seasonal|\|federal\||gobierno/)
-  SIMPLE_SINGULARS = %w{
-    statistic number level rate description trend growth projection survey forecast figure report verification record
-    authorization card classification form hazard poster fair board outlook grant funding factor other cut
+  SIMPLE_SEARCHES = '(job|employment|internship)s?'.freeze
+  JOB_RELATED_KEYWORDS = '((position|opening|posting|job|employment|career|' \
+                         'intern(ship)?|seasonal|trabajo|puesto|empleo|' \
+                         'carrera|vacante+?)(s\b|\b)|(opportunit|vacanc)' \
+                         '(y|ies))|(posicion|ocupacion|oportunidad|federal)'\
+                         '(es)?|gobierno'.
+                         freeze
```

Comment:
@peggles2 , it would be cleaner to use `# frozen_string_literal: true` at the top of the file than to use `freeze` on all the strings.